### PR TITLE
Fix usage of zadd

### LINKED ISSUE #73: Wrong signature for zadd
Predis ClientInterface declares zadd as: 

https://github.com/nrk/predis/blob/111d100ee389d624036b46b35ed0c9ac59c71313/src/ClientInterface.php#L116
` * @method int    zadd($key, array $membersAndScoresDictionary)`

While RedisMock declares:
https://github.com/M6Web/RedisMock/blob/f4c6f55497becb1c6f5d9593b2a74163e0281858/src/M6Web/Component/RedisMock/RedisMock.php#L931
` public function zadd($key, $score, $member) {`

Am I missing something?

### PR DESCRIPTION
**Pull request by magnetik**
Allow zadd to have 2 arguments or an array as first argument

Fixes #73

### CODE REVIEW

File: src/M6Web/Component/RedisMock/RedisMock.php [hunk 1]
```diff
@@ -943,9 +943,15 @@ public function zrevrangebyscore($key, $max, $min, array $options = array())
...
-            throw new UnsupportedException('In RedisMock, `zadd` command can not set more than one member at once.');
+    public function zadd($key, ...$args) {
+        if (count($args) === 1) {
+            $score = reset($args[0]);
+            $member = key($args[0]);
+        } elseif (count($args) === 2) {
+            $score = $args[0];
+            $member = $args[1];
+        } else {
+            throw new UnsupportedException('In RedisMock, `zadd` command blabla:)');
```
Comment (omansour) [inline src/M6Web/Component/RedisMock/RedisMock.php]: txs. please add a meaningful error message

Comment (magnetik) [reply src/M6Web/Component/RedisMock/RedisMock.php]: oh right, I'm adding a test too

**Comment by omansour** [comment]
v4.4.0

**Comment by magnetik** [comment]
For some reason it seems that v4.4.0 is not published automatically on packagist. Maybe something is wrong in the github <> packagist side?

**Pull request merged**